test/Datadog.Trace.ClrProfiler.IntegrationTests/MsmqTests.cs

```diff
@@ -0,0 +1,111 @@
+#if NETFRAMEWORK
+using System;
+using System.Linq;
+using Datadog.Core.Tools;
+using Datadog.Trace.TestHelpers;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
+{
+    public class MsmqTests : TestHelper
+    {
+        private const string ExpectedServiceName = "Samples.Msmq-msmq";
+
+        public MsmqTests(ITestOutputHelper output)
+            : base("Msmq", output)
+        {
+            SetServiceVersion("1.0.0");
+        }
+
+        [Trait("Category", "EndToEnd")]
```

Comment:
```suggestion
        [Trait("Category", "EndToEnd")]
        [Trait("RunOnWindows", "True")]
```

Sorry this isn't intuitive, but the `RunOnWindows=true` is a filter we use when deciding which integration tests to run on Windows